lib/engine.js

```diff
@@ -292,7 +293,11 @@ module.exports = function container (get, set, clear) {
               }
               if (order.status === 'rejected' && (order.reject_reason === 'post only' || api_order.reject_reason === 'post only')) {
                 msg('post-only ' + type + ' failed, re-ordering')
-                return cancelOrder(true)
+                return cb(null, null)
+              }
+              if (order.status === 'rejected' && order.reject_reason === 'balance') {
```

Comment:
See above. Why check `order.status` (which was copied from `api_order`) and then check `api_order`? This is not consistent.

Since we do not agree, a decision by a maintainer, i.e. @DeviaVir or @carlos8f, on this one and line 285 are needed.